scripts/mempool.js

```diff
@@ -236,49 +244,25 @@ export class Mempool {
                 if ((UTXO_STATE & filter) == 0) {
                     continue;
                 }
+                if (!includeLocked && wallet.isCoinLocked(vout.outpoint)) {
+                    continue;
+                }
                 totBalance += vout.value;
             }
         }
         return totBalance;
     }
-    /**
-     * Outpoint that we want to fetch
-     * @param {COutpoint} op
-     * @param {UTXO_WALLET_STATE} filter the filter you want to apply
-     * @param {Boolean} onlyConfirmed consider only confirmed transactions
-     */
-    hasUTXO(op, filter, onlyConfirmed) {
```

Comment:
Should we remove this? The only alternative to check if a UTXO exists in this case, would be calling `getUTXOs` and checking for our TXID and COutpoint, or checking `mempool.txmap`, could make future code more verbose than necessary.

Reply:
I removed it because at the moment it is unused and I don't see a scenario where we need it. I'd say if it is needed in a future PR we can re-add it